mindmeld/components/classifier.py

```diff
@@ -346,6 +345,7 @@ def _get_model_config(loaded_config=None, **kwargs):
         """
         try:
             # If all params required for model config were passed in, use kwargs
+            kwargs['tokenizer'] = self._resource_loader.query_factory.tokenizer
```

Comment:
This is silently overwriting a tokenizer that gets passed in

Reply:
resolved